common/views/components/SearchBar/SearchBar.tsx

```diff
@@ -64,6 +65,17 @@ const SearchBar: FunctionComponent<{ type: string }> = ({ type }) => {
             placeholder="search..."
             ariaLabel={`Search ${type}`}
           />
+          {inputQuery && (
+            <ClearSearch
+              inputRef={searchInput}
+              setValue={setInputQuery}
+              gaEvent={{
```

Comment:
Again, is this something we'd want to track 🤔 Do we have a list somewhere of these?